examples/local/teardown.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -x
+
+. ./env.sh
+
+kubectl delete aws -l clusterID=${CLUSTER_NAME}
+
+kubectl delete -f configmap.yaml
+kubectl delete -f configmap-ssh.yaml
+kubectl delete -f deployment.yaml
```

Comment:
By this time maybe the cluster hasn't been deleted yet, removing the operator can leave resources behind. It would be nice to be able to wait until the cluster is completely deleted before removing the deployment.

Reply:
Completely right, just happened yesterday after pushing my PR :)

Reply:
Yes it's definitely better to wait until all the AWS resources have been deleted. There is the log line for `cluster 'xxxxx' deleted` which would work but sadly I can't think of a better way of detecting this.